client/components/ThingsToKnowComponents/ThingsToKnow.jsx

```diff
@@ -8,17 +8,19 @@ import HouseRulesModal from '../Modals/HouseRulesModal.jsx';
 import CancellationModal from '../Modals/CancellationModal.jsx';
 import SafetyModal from '../Modals/SafetyModal.jsx';
 
-const Container = styled.div`
+// Width is no 100% to inherite the 60% from its parent element
+const ThingsToKnowContainer = styled.div`
   display: flex;
```

Comment:
when I redo my css, I am going to apply this!

Reply:
Yesss.